### PR TITLE
Homework2: Mila Bendina

### PR DESCRIPTION
**Pull request by milabendina**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle).
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks.
- [ ] my PR made from the fork feature branch to appropriate branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

**Comment by milabendina** [comment]
there are only 3 ready tasks in the 8th task, 2 left I'll do before the deadline

File: src/homework2/task2.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.
+Подсказки:
+my_string.split([chars]) возвращает список строк.
+len(list) - количество элементов в списке.
+'''
+
+
+sent = input('Введите предложение: ')
+punctuation_marks = '.,-;:?!'
```
Comment (belovmd) [inline src/homework2/task2.py]: what about `)([]` and others

File: src/homework2/task4.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+str_ = input('Введите строку: ')
+small = 0
+big = 0
+
+for symbol in str_:
+    if 'a' <= symbol <= 'z':
+        small += 1
+    else:
```
Comment (belovmd) [inline src/homework2/task4.py]: `elif`

File: src/homework2/task5.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+циклические операторы и условные операторы. n - вводится.
+'''
+
+
+n = int(input('Введите n: '))
+n0 = 0
+n1 = 1
+
+while n > 0:
+    n2 = n1
```
Comment (belovmd) [inline src/homework2/task5.py]: a, b = a+b, a

File: src/homework2/task6.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+Число положительное целое, произвольной длины. Задача требует работать только с числами
+(без конвертации числа в строку или что-нибудь еще).
+'''
+
+
+num = int(input('Введите число: '))
+rev = 0
+
+while num > 0:
+    rev = (rev * 10) + (num % 10)
```
Comment (belovmd) [inline src/homework2/task6.py]: redundant parentheses

File: src/homework2/task7.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+side_1 = int(input('Первая сторона: '))
+side_2 = int(input('Вторая сторону: '))
+side_3 = int(input('Третьяя сторона: '))
+p = side_1 + side_2 + side_3
+
+if side_1 + side_2 <= side_3:
+    print('Фигура не является треугольником.')
+else:
+    print('S = ', (p * (p - side_1) + (p - side_2) + (p - side_3)) ** 0.5)
```
Comment (belovmd) [inline src/homework2/task7.py]: wrong formula, doesn't solve task in description

File: src/homework2/task7.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+Если нет, вывести сообщение о неверных данных.
+'''
+
+
+side_1 = int(input('Первая сторона: '))
+side_2 = int(input('Вторая сторону: '))
+side_3 = int(input('Третьяя сторона: '))
+p = side_1 + side_2 + side_3
+
+if side_1 + side_2 <= side_3:
```
Comment (belovmd) [inline src/homework2/task7.py]: not all necessary checks

**Review by belovmd — CHANGES_REQUESTED** [review]
you can add one more advanced task for higher score

File: src/homework2/task2.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.
+Подсказки:
+my_string.split([chars]) возвращает список строк.
+len(list) - количество элементов в списке.
+'''
+
+
+sent = input('Введите предложение: ')
+punctuation_marks = '.,-;:?!"@№#$%^&*'
```
Comment (belovmd) [inline src/homework2/task2.py:11]: string.punctuation

**Review by belovmd — COMMENTED** [review]
missed 1 advanced task
8,5

**Pull request closed without merging**